examples/sparse_gamma_def.py

```diff
@@ -0,0 +1,153 @@
+# this is an implementation of the sparse gamma deep exponential family experiment presented in
+# Ranganath, Rajesh, Tang, Linpeng, Charlin, Laurent, and Blei, David. Deep exponential families.
+#
+# the Olivetti faces dataset is originally from http://www.cl.cam.ac.uk/research/dtg/attarchive/facedatabase.html
+#
+# compare to the implementation here:
+# https://github.com/blei-lab/ars-reparameterization/tree/master/sparse%20gamma%20def
+
+
+from __future__ import absolute_import, division, print_function
+
+import argparse
+import os
+
+import numpy as np
+import torch
+from torch import nn as nn
+
+import pyro
+import pyro.optim as optim
+import pyro.poutine as poutine
+import wget
```

Comment:
can you add this to the extra_requires in setup.py

Reply:
ok